de-lib/src/main/java/org/iplantc/de/apps/client/AppDetailsView.java

```diff
@@ -40,6 +40,8 @@
                                         DetailsCategoryClicked.HasDetailsCategoryClickedHandlers {
 
     interface AppDetailsAppearance {
+        String imageLabel();
```

Comment:
Really really minor, but this is the only variable above the `AppDetailsStyle` interface so it looks a bit out of place.